.travis.yml

```diff
@@ -10,6 +10,10 @@ before_script:
   - cp keys/saml.key.enc.example keys/saml.key.enc
   - bin/rake db:setup --trace
   - bin/rake assets:precompile
+after_script:
+  - npm install -g pa11y-crawl
+  - bundle exec rails s -d
+  - pa11y-crawl --standard Section508 --ci http://localhost:3000
```

Comment:
No not just make these part of `script:`?

We already have a npm install in the before section, perhaps we should just add pa11y-crawl to the package.json (can you have dev dependecies in package.json?)?

With the addition of this extra step we've almost got enought steps to warrent a /bin/travis_run.sh script.   There are two main benefits to have such as script:
1) local devs can reproduce all the CI steps with a single command
2) We can configure it so that it will fail when a given step fails.  One downsite to the `script:` section is that travis will continue to run fugure steps even if one of them fails.